sage-ios/SAGE/SAGE/CheckinRequestTableViewCell.swift

```diff
@@ -18,12 +18,14 @@ class CheckinRequestTableViewCell: UITableViewCell {
     var checkButton = UIButton()
     var xButton = UIButton()
     
+    var checkinID: Int?
```

Comment:
actually i dont end up using this variable anywhere so i can just delete it

(in the tableview controller, you get a section index when something is clicked so you can access the specific checkin by accessing the array at that index)